src/main/java/org/candlepin/model/ContentAccessCertificateCurator.java

```diff
@@ -218,4 +218,28 @@ public List<CertSerial> listCertSerials(Collection<String> consumerIds) {
         return serials;
     }
 
+    /**
+     * Retrieves all of the content access certificate IDs for all of the provided {@link Consumer}s.
+     *
+     * @param consumerUuids
+     *  the UUIDs of the consumers to retrieve the certificate IDs for
+     *
+     * @return the content access certificate IDs for all of the provided consumers
+     */
+    public List<String> getIdsForConsumers(Collection<String> consumerUuids) {
+        if (consumerUuids == null || consumerUuids.isEmpty()) {
+            return new ArrayList<>();
+        }
+
+        String jpql = "SELECT c.contentAccessCert.id FROM Consumer c WHERE c.uuid IN (:uuids)";
+        Query query = entityManager.get().createQuery(jpql, String.class);
```

Comment:
Let's use the `getEntityManager` method instead of hitting the provider directly.

Reply:
Updated